atomic_reactor/plugins/pre_add_yum_repo.py

```diff
@@ -33,7 +33,8 @@ def __init__(self, tasker, workflow, repo_name, baseurl):
         """
         # call parent constructor
         super(AddYumRepoPlugin, self).__init__(tasker, workflow)
-        self.repo = YumRepo(baseurl=baseurl, name=repo_name, enabled=True)
+        self.repo_name = repo_name
```

Comment:
those 2 new lines are missing coverage

Reply:
This whole class is not tested :disappointed: 
Unrelated to this change though